scripts/kustomize/kubeadm/init-orig/kubeadm-cluster-config-v1beta2.yml

```diff
@@ -7,7 +7,7 @@ kubernetesVersion: v$KUBERNETES_VERSION
 certificatesDir: /etc/kubernetes/pki
 clusterName: kubernetes
 controllerManager: {}
-useHyperKubeImage: true
+useHyperKubeImage: false
```

Comment:
Is this going to work for 1.15 airgap installs that still have hyperkube in the airgap pacakge?

Reply:
I think the best solution for me here is to remove hyperkube for the 1.15 installs as well

Reply:
changed